src/store/ChartStore.js

```diff
@@ -339,6 +341,7 @@ class ChartStore {
     }
 
     onMarketOpenClosedChange = (changes) => {
+        this.loader.setState('trading-time');
```

Comment:
@shayanbinary And also about `trading-time`, why you put it here? shouldnt it be in `_updateTradeTimes`.